src/unix/mod.rs

```diff
@@ -230,7 +230,7 @@ pub const INADDR_BROADCAST: in_addr_t = 4294967295;
 pub const INADDR_NONE: in_addr_t = 4294967295;
 
 cfg_if! {
-    if #[cfg(dox)] {
+    if #[cfg(any(dox, target_os = "l4re"))] {
```

Comment:
Could you separate this out explicitly to a separate clause below? (just to be clear that this branch is the "doc only" branch.